pkgs/development/python-modules/brotli/default.nix

```diff
@@ -1,38 +1,41 @@
 {
   lib,
+  brotli,
   buildPythonPackage,
   fetchFromGitHub,
+  pkgconfig,
   pytestCheckHook,
   setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "brotli";
-  version = "1.1.0";
+  inherit (brotli) version src;
   pyproject = true;
 
-  src = fetchFromGitHub {
-    owner = "google";
-    repo = "brotli";
-    tag = "v${version}";
-    hash = "sha256-U1vAupUthD5W0xvlOKdgm9MAVLqsVyZUaFdeLsDAbDM=";
-    # .gitattributes is not correct or GitHub does not parse it correct and the archive is missing the test data
```

Comment:
Note that since we no longer have the test data, the number of tests run reduces from 595 to 135.
We cannot set `forceFetchGit = true` in the brotli expression because that causes an infinite recursion, I assume via git.

Reply:
But then `fetchFromGitHub` wouldn't work even without `forceFetchGit = true`, would it?

Reply:
I cannot reproduce this locally. ~~May it have been caused by fetchpatch?~~ Something unrelated must have changed here 🤔

Reply:
I can still reproduce this. Did you change the hash to `lib.fakeHash` or something else that's invalid?